### DISCUSSION TITLE
请问是否有设计资源呢

### DISCUSSION THREAD

**Discussion by yangfancuanyuxi** [Moonofweisheng/wot-design-uni#373] [Q&A]
我们希望可以基于wot-design-uni进行页面设计和开发，请问可以下载组件的Sketch/Axure或其他形式的设计资源吗

**Comment by Moonofweisheng** [comment]
设计资源在计划中，目前还没有提供，计划使用即使设计来做，不过我也不是专业UI，所以需要边学边做，现在比较忙还没什么时间搞。




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "Moonofweisheng/wot-design-uni"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年6月17日(星期一) 下午2:22
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[Moonofweisheng/wot-design-uni] 请问是否有设计资源呢 (Discussion #373)





 
我们希望可以基于wot-design-uni进行页面设计和开发，请问可以下载组件的Sketch/Axure或其他形式的设计资源吗
